### PR TITLE
Added PostgreSQL SSL/TLS Client Certificate support

### PR DESCRIPTION
**Pull request by PeterVanPansen**
+ New auth_opt_* option --> auth_opt_sslcert
+ New auth_opt_* option --> auth_opt_sslkey
+ Added new postgresql options to README
+ Corrected topic-permission statement for postgresql in README

### CODE REVIEW

File: be-postgres.c [hunk 1]
```diff
@@ -90,22 +94,31 @@ void *be_pg_init()
...
+	conf->sslkey = sslkey;
 
 	_log(LOG_DEBUG, "HERE: %s", conf->superquery);
 	_log(LOG_DEBUG, "HERE: %s", conf->aclquery);
 
-
-	char *connect_string = NULL;
-
-	conf->conn = PQsetdbLogin(conf->host, conf->port, NULL, NULL, conf->dbname, conf->user, conf->pass);
+	char conninfo[1024] = "";
```
Comment (jpmens) [inline be-postgres.c]: please increase the size of `conninfo`; paths to keys/certs can be quite long

File: be-postgres.c [hunk 2]
```diff
@@ -90,22 +94,31 @@ void *be_pg_init()
...
+	sprintf(conninfo,
```
Comment (jpmens) [inline be-postgres.c]: use snprintf()

**Comment by jpmens** [comment]
Merged, thank you.

**Comment by PeterVanPansen** [comment]
Thank you for the great plugin!

**Pull request merged**